python-project-template/.github/workflows/{% if include_benchmarks %}asv-pr.yml{% endif %}.jinja

```diff
@@ -1,101 +1,86 @@
-
-      - name: Get current job logs URL
-        uses: Tiryoh/gha-jobid-action@v0
-        id: jobs
-        with:
-          github_token: {% raw %}${{ secrets.GITHUB_TOKEN }}{% endraw %}
-          job_name: {% raw %}${{ github.job }}{% endraw %}
-
-      - name: Create ASV machine config file
-        run: asv machine --machine gh-runner --yes
-
-      - name: Run comparison of PR against main branch
-        run: |
-          git remote add upstream https://github.com/{% raw %}${{ github.repository }}{% endraw %}.git
-          git fetch upstream
-          asv continuous upstream/main HEAD --verbose || true
-          asv compare upstream/main HEAD --sort ratio --verbose | tee output
-          python -m lf_asv_formatter --asv_version "$(echo asv --version)"
-          printf "\n\nClick [here]($STEP_URL) to view all benchmarks." >> output
-        env:
-          STEP_URL: "{% raw %}${{ steps.jobs.outputs.html_url }}{% endraw %}#step:8:1"
-
-      - name: Find benchmarks comment
-        uses: peter-evans/find-comment@v2
-        id: find-comment
-        with:
-          issue-number: {% raw %}${{ github.event.pull_request.number }}{% endraw %}
-          comment-author: 'github-actions[bot]'
-          body-includes: view all benchmarks
-
-      - name: Create or update benchmarks comment
-        uses: peter-evans/create-or-update-comment@v3
-        with:
-          comment-id: {% raw %}${{ steps.find-comment.outputs.comment-id }}{% endraw %}
-          issue-number: {% raw %}${{ github.event.pull_request.number }}{% endraw %}
-          body-path: {% raw %}${{ env.WORKING_DIR }}/output{% endraw %}
-          edit-mode: replace
\ No newline at end of file
+    - name: Checkout PR branch of the repository
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: Display Workflow Run Information
+      run: |
+        echo "Workflow Run ID: ${{ github.run_id }}"
+    - name: Cache Python ${{ env.PYTHON_VERSION }}
+      uses: actions/cache@v3
+      with:
+        path: ~/.cache/pip
+        key: python-${{ env.PYTHON_VERSION }}
+    - name: Install dependencies
+      run: |
+        sudo apt-get update
+        python -m pip install --upgrade pip
+        pip install asv==0.6.1 virtualenv tabulate lf-asv-formatter
+    - name: Make artifacts directory
+      run: mkdir -p ${{ env.ARTIFACTS_DIR }}
+    - name: Save pull request number
+      run: echo ${{ github.event.pull_request.number }} > ${{ env.ARTIFACTS_DIR }}/pr
+    - name: Get current job logs URL
+      uses: Tiryoh/gha-jobid-action@v0
+      id: jobs
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        job_name: ${{ github.job }}
+    - name: Create ASV machine config file
+      run: asv machine --machine gh-runner --yes
+    - name: Save comparison of PR against main branch
+      run: |
+        git remote add upstream https://github.com/${{ github.repository }}.git
```

Comment:
I think I missing the line that tells `publish-benchmarks-pr.yml` to run when this workflow is executed in a fork. I believe it happens, I just don't immediately see the plumbing that makes it work.

Reply:
I added a pointer to the Github documentation I used to create this implementation!